pip/src/interpreter.rs

```diff
@@ -106,6 +106,12 @@ pub(crate) enum TargetProfile {
     /// capabilities, as well as the optional integer computation and qubit
     /// reset capabilities, as defined by the QIR specification.
     Adaptive_RI,
+    /// Target supports the Adaptive profile with integer & floating point computation and qubit reset capabilities.
+    ///
+    /// This profile includes all of the required Adaptive Profile and Adaptive_RI
+    /// capabilities, as well as the optional floating point computation and qubit
+    /// reset capabilities, as defined by the QIR specification.
```

Comment:
As prior comment, including `Adaptive_RI` already includes reset capabilities.

Reply:
Reworded them. The adaptive profile requires qubit reset now, so I updated the adaptive RI docs as well.